ert_gui/main_window.py

```diff
@@ -70,6 +76,18 @@ def __createMenu(self):
         show_about.setMenuRole(QAction.ApplicationSpecificRole)
         show_about.triggered.connect(self.__showAboutMessage)
 
+        help_links = {}
+        with pkg_resources.resource_stream('ert_gui', os.path.join("resources", "gui", "help", "help_links.yml")) as stream:
+            try:
+                help_links = yaml.safe_load(stream)
+            except yaml.YAMLError as exc:
+                print("Warning: Unable to load help links configuration file.")
```

Comment:
this is much better, but the exception itself should also be printed (it containts infor about what the error is). You could use `traceback.print_exc` 
(normally I'd suggest to print the warning to `logging.warning` and the exception to `logging.debug` but since no logging system has been set up yet in ERT, let's just keep using `print` for now)

Reply:
Why not crash and burn? It is clearly not very critical that this works, but at the same time it indicates a serious deploy problem with faulty internal package data. I see no reason why we should not crash hard. In particular that will make it quicker to pick this up when doing end to end testing of _ERT_.